pkg/state/type.go

```diff
@@ -28,6 +28,7 @@ type rootState struct {
 	Interfaces  []interfaceState `json:"interfaces"             yaml:"interfaces"`
 	Routes      *routes          `json:"routes,omitempty"       yaml:"routes,omitempty"`
 	DNSResolver *dnsResolver     `json:"dns-resolver,omitempty" yaml:"dns-resolver,omitempty"`
+	Ovn         *bridgeMappings  `json:"ovn,omitempty"          yaml:"ovn,omitempty"`
```

Comment:
```suggestion
	Ovn         json.RawMessage  `json:"ovn,omitempty"          yaml:"ovn,omitempty"`
```

Reply:
Now that we are a it maybe we can add the missing ones looking at https://docs.rs/nmstate/latest/nmstate/struct.NetworkState.html

We are missing:
- hostname
- route-rules
- ovs-db

So we can do the following
```golang
type rootState struct {
	Interfaces  []interfaceState `json:"interfaces"             yaml:"interfaces"`
	Routes      *routes          `json:"routes,omitempty"       yaml:"routes,omitempty"`
	DNSResolver *dnsResolver     `json:"dns-resolver,omitempty" yaml:"dns-resolver,omitempty"`
        Hostname         json.RawMessage  `json:"hostname,omitempty"          yaml:"ovn,omitempty"`
        RouteRules         json.RawMessage  `json:"route-rules,omitempty"          yaml:"ovn,omitempty"`
        OVSDb         json.RawMessage  `json:"ovs-db,omitempty"          yaml:"ovs-db,omitempty"`
        OVN         json.RawMessage  `json:"ovn,omitempty"          yaml:"ovn,omitempty"`
}
```

Reply:
An alternative is a custom encoder that will just bypass whatever we don't need to filter

Reply:
I'd rather focus exclusively on the attributes that I need to add. And test only those attributes.